src/main/java/io/jsonwebtoken/InvalidClaimException.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2015 jsonwebtoken.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.jsonwebtoken;
+
+/**
+ * InvalidClaimException is a subclass of the {@link ClaimJwtException} that is thrown after a validation of an JTW claim failed.
```

Comment:
```java
/**
 * Exception indicating a parsed claim is invalid in some way.  Subclasses reflect the specific 
 * reason the claim is invalid.
 *
 * @see IncorrectClaimException
 * @see MissingClaimException
 */
```

For abstract classes or intermediate classes in public APIs, I try to show or link to other examples that give the class more contextual relevance.  Using the `@see` jdoc annotation is a simple/useful way to do this.